.gitattributes

```diff
@@ -0,0 +1,8 @@
+* text=auto
+
+yarn.lock linguist-generated=false
+
+# yarn v3
```

Comment:
Not sure we should merge this as-is given that this is for Yarn v3, but this repo still uses Yarn v1. Likewise for the gitignore yarn entries.

Reply:
I'm about to upgrade Yarn, but sure, I can leave this out to keep the commit clean.